scripts/prepare_sector_network.py

```diff
@@ -70,6 +70,17 @@ def define_spatial(nodes, options):
 
     spatial.biomass.df = pd.DataFrame(vars(spatial.biomass), index=nodes)
 
+    # unsustainable bioliquids
+    spatial.bioliquids = SimpleNamespace()
+    if options.get("regional_oil_demand"):
```

Comment:
Shouldn't this adopt the same logic as `spatial.biomass`?
```suggestion
    if options.get("biomass_spatial", options["biomass_transport"]):
```
Or could integrate as `spatial.biomass.bioliquids = nodes + " bioliquids"`